state/prune.go

```diff
@@ -208,16 +217,19 @@ func (p *collectionPruner) deleteInBatches(iter *mgo.Iter, logTemplate string, s
 
 			now := time.Now()
 			if now.Sub(lastUpdate) >= historyPruneProgressSeconds*time.Second {
```

Comment:
Do we care about the historyPruneProgressSeconds value?

Reply:
I would say that if the deletion takes more than 15s in total, yes we do.